internal/juju/offers.go

```diff
@@ -74,6 +75,12 @@ type RemoveRemoteOfferInput struct {
 	OfferURL  string
 }
 
+type GrantRevokeOfferInput struct {
+	User     string
+	Access   string
+	OfferURL string
```

Comment:
I think this should be URL's, you can revoke many at once

Reply:
Since we discussed changing the schema in the last terraform office hours, I think this no longer apply, WDYT?